src/utils/core.js

```diff
@@ -0,0 +1,3 @@
+// Replaces {key} with data in a string template
+export const setTemplate = (text, data) =>
```

Comment:
Could you create a couple of unit tests for this function?

Reply:
Added.